x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/40_engine_delete.yml

```diff
@@ -26,6 +26,12 @@ setup:
       engine.get:
         engine_id: test-engine-to-delete
 
+  - do:
+      indices.exists_alias:
+        name: engine-test-engine-to-delete
+
+  - is_false: ''
```

Comment:
Curious - what does this do, check that the response of the previous invocation is false?

Reply:
That's correct!! 

I changed this to `is_true` to run it locally, and this is the output you'll get if the value isn't correct: 
```
java.lang.AssertionError: Failure at [entsearch/40_engine_delete:40]: field [] doesn't have a true value
Expected: not a string equal to "false" ignoring case
     but: was "false"
```